www/front_src/src/Resources/Graph/Performance/Legend/index.tsx

```diff
@@ -97,29 +131,47 @@ const LegendContent = ({
             onSelect(metric);
           }}
         >
-          {name}
+          {getFormattedName({
+            name,
+            unit,
+          })}
         </Typography>
       </div>
     );
   };
 
   return (
-    <>
+    <div className={classes.items}>
       {lines.map((line) => {
         const { color, name, display } = line;
 
         const markerColor = display
           ? color
           : fade(theme.palette.text.disabled, 0.2);
 
+        const metric = find(
+          equals(line.metric),
+          propOr([], 'metrics', metricsValue),
+        );
+
+        const formattedValue =
+          metric && getFormattedMetricData(metric)?.formattedValue;
+
         return (
           <div className={classes.item} key={name}>
             <LegendMarker disabled={!display} color={markerColor} />
-            {getLegendName(line)}
+            <div className={classes.legendData}>
+              {getLegendName(line)}
```

Comment:
We need to handle the instance, which corresponds to any prefix ending with the character # (for example: MyCentreonInstance#MetricName). The instance should be displayed when the legend is hovered, as well as the complete legend name (MyCentreonInstance & MetricName for the latter)